src/main/java/ru/itmo/roguelike/items/Armor.java

```diff
@@ -0,0 +1,27 @@
+package ru.itmo.roguelike.items;
+
+import ru.itmo.roguelike.characters.Actor;
+import ru.itmo.roguelike.render.particles.MovingUpText;
+
+import java.awt.*;
+
+public class Armor extends Collectible {
```

Comment:
It better be an abstract class with concrete implementations. All such implementations should have it's own protections points.

Also it's cool to have a name for such instances, like "Leather Jacket" or "Steel Breastplate" to show as text when used